metpy/calc/tests/test_kinematics.py

```diff
@@ -470,3 +471,32 @@ def test_storm_relative_helicity():
     assert_almost_equal(p_srh, srh_true_p, 2)
     assert_almost_equal(n_srh, srh_true_n, 2)
     assert_almost_equal(T_srh, srh_true_t, 2)
+
+
+def test_v_vorticity():
+    """Test that v_vorticity wrapper works (deprecated in 0.7)."""
+    a = np.arange(3)
+    u = np.c_[a, a, a] * units('m/s')
+    v = v_vorticity(u, u, 1 * units.meter, 1 * units.meter, dim_order='xy')
+    true_v = np.ones_like(u) / units.sec
+    assert_array_equal(v, true_v)
+
+
+def test_convergence():
+    """Test that convergence wrapper works (deprecated in 0.7)."""
+    a = np.arange(3)
+    u = np.c_[a, a, a] * units('m/s')
+    c = convergence(u, u, 1 * units.meter, 1 * units.meter, dim_order='xy')
+    true_c = np.ones_like(u) / units.sec
+    assert_array_equal(c, true_c)
+
+
+def test_convergence_vorticity():
+    """TTest that convergence_vorticity wrapper works (deprecated in 0.7)."""
```

Comment:
You stuttered here. 😁

Reply:
![](https://media.giphy.com/media/gnJgBlPgHtcnS/giphy.gif)